apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -80,7 +99,14 @@ function SectionProgressSelector({
   // If the user has not selected manually the v1 or v2 table, show the DCDO defined default.
   // If a user has selected manually, show that version.
   const isPreferenceSet = showProgressTableV2 !== undefined;
-  const displayV2 = isPreferenceSet
+  const params = queryParams('view');
+
+  // If there is a url pram, use that param to determine to show V2.
+  const displayV2FromUrl = params === 'v2';
+
+  const displayV2 = displayV2FromUrl
+    ? true
+    : isPreferenceSet
     ? showProgressTableV2
     : DCDO.get('progress-table-v2-default-v2', false);
```

Comment:
```suggestion
  const displayV2 = displayV2FromUrl || (
    isPreferenceSet
    ? showProgressTableV2
    : DCDO.get('progress-table-v2-default-v2', false));
```

I think this is slightly clearer, but another optional nit.